visualization_msgs/msg/Marker.msg

```diff
@@ -56,6 +56,11 @@ geometry_msgs/Point[] points
 # NOTE: alpha is not yet used
 std_msgs/ColorRGBA[] colors
 
+# An image to be loaded into the rendering engine as the texture for this marker.
+sensor_msgs/Image texture
```

Comment:
Is this the right type to use here? Doesn't this require the texture to be an uncompressed image? Is that a reasonable restriction? We don't need to do image processing on the texture (as far as I know), we're just passing it to the 3D engine to render.

Furthermore, I don't think we don't need a `frame_id` or arguably the timestamp from the `header`, nor the `width` or `height`.

I think `sensor_msgs/CompressedImage` is probably a better fit:

https://docs.ros.org/en/api/sensor_msgs/html/msg/CompressedImage.html

Reply:
:+1:  Switching to compressed makes sense for this.